modules/mod-deep-learning/ModelManagerPanel.h

```diff
@@ -0,0 +1,88 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ModelManagerPanel.h
+   Hugo Flores Garcia
+
+******************************************************************/
+/**
+
+\class ModelManagerPanel
+\brief ModelManagerPanel TODO
+
+*/
+/*******************************************************************/
+
+#pragma once
+
+#include "DeepModelManager.h"
+#include "ModelCardPanel.h"
+
+int getScreenWidth(float scale = 1);
+int getScreenHeight(float scale = 1);
+wxSize getManagerToolsPanelSize();
+
+class DeepLearningEffectBase;
+class ShuttleGui;
+class ModelManagerPanel;
+
+static const int cardPanel_w = 600;
+static const int cardPanel_h = 150;
+static const int cardPanel_x_offset = 20;
+
+static const int detailedCardPanel_w = 400;
+static const int detailedCardPanel_h = 400;
+
+static const int managerPanel_w = cardPanel_w + detailedCardPanel_h + 
+                                   cardPanel_x_offset + 20;
+
+class ManagerToolsPanel : public wxPanelWrapper
+{
+public:
+   
+   ManagerToolsPanel(wxWindow *parent, ModelManagerPanel *panel);
+
+   void PopulateOrExchange(ShuttleGui & S);
+
+   void SetFetchProgress(int64_t current, int64_t total);
+   void OnAddRepo(wxCommandEvent & WXUNUSED(event));
+   void OnExplore(wxCommandEvent & WXUNUSED(event));
+
+private:
+   wxStaticText *mFetchStatus;
+   wxButton *mAddRepoButton;
+   wxButton *mExploreButton;
+   ModelManagerPanel *mManagerPanel;
```

Comment:
Would be nice to add `{}` initializers

Reply:
4619c09faa3e0b2f4f54f70adb7e3b8721b4e646